cvat-core/src/server-proxy.js

```diff
@@ -595,6 +595,11 @@
                 }
 
                 const taskData = new FormData();
+
+                const clientFile = taskDataSpec.client_files[0];
+                const clientFiles = taskDataSpec.client_files;
+                const uploadUsingChunks = clientFiles.length === 1;
```

Comment:
I would suggest also checking if the file size more than chunkSize you use.

Reply:
Done